src/types/ChatRequest.ts

```diff
@@ -21,3 +28,10 @@ export type ChatRequest = {
   fromJoined: boolean;
   toJoined: boolean;
 };
+
+export interface VideoChatRequest {
+  hostId: string;
+  guestId: string;
+  state: VideoChatRequestState;
+  createdAt: number;
+}
```

Comment:
At this point, we don't need this kind of additional information. It would be extremely easy to add these fields in the `api` functions. I'm going to use this `createdAt` for expiration logic